mypy/stubgenc.py

```diff
@@ -237,6 +237,10 @@ def is_c_classmethod(obj: object) -> bool:
     )
 
 
+def is_c_staticmethod(obj: object) -> bool:
+    return type(obj).__name__ in ("staticmethod")
```

Comment:
Probably this is not what you mean, I guess it should be:

```suggestion
    return type(obj).__name__ in ("staticmethod",)
```

Reply:
Or just:

```suggestion
    return type(obj).__name__ == "staticmethod"
```

Reply:
You are right, I will simplify this code and use @AlexWaygood's approach.
I originally copied the code from #13574, where the author probably tried to mirror the implementation of `is_c_classmethod`.

Reply:
Hi, is there any update on this PR? Can it be merged or is there still something I need to improve?